datasets/datasets_openml.py

```diff
@@ -2,31 +2,34 @@
 
 with safe_import_context() as import_ctx:
     import openml
+    import pandas as pd
 
 
 DATASETS = {
-    'bank-marketing': 44126,
-    'california': 44090,
-    'compass': 44162,
-    'covertype': 44121,
-    'covertype_categorical': 44159,
-    'credit': 44089,
-    'electricity': 44120,
-    'electricity_categorical': 44156,
-    'eyemovements': 44130,
-    'eyemovements_categorical': 44156,
-    'higgs': 44129,
-    'house_16H': 44123,
-    'jannis': 44131,
-    'KDDCup06_upselling': 44186,
-    'kdd_ipums_la_97-small': 44124,
-    'magictelescope': 44125,
-    'MiniBooNE': 44128,
-    'phoneme': 44127,
-    'pol': 44122,
-    'rl': 44160,
-    'road_safety': 44161,
-    'wine': 44091,
+        44126,  # bank-marketing
+        44090,  # california
+        # 44162,  # compass
+        44121,  # covertype
+        # 44159,  # covertype_categorical
+        44089,  # credit
+        # 151,  # electricity
+        44120,  # electricity
+        # 44156,  # electricity_categorical
+        44130,  # eyemovements
+        # 44156,  # eyemovements_categorical
+        44129,  # higgs
+        44123,  # house_16H
+        44131,  # jannis
+        # 44186,  # KDDCup06_upselling
+        44124,  # kdd_ipums_la_97-small
+        44125,  # magictelescope
+        44128,  # MiniBooNE
+        44127,  # phoneme
+        44122,  # pol
+        # 44160,  # rl
+        # 44161,  # road_safety
+        # 42803,  # road-safety
+        44091,  # wine
```

Comment:
The proper fix is not this one as it makes it very hard to know which dataset is which.
```suggestion
    'bank-marketing': 44126,
    'california': 44090,
    'compass': 44162,
    'covertype': 44121,
    'covertype_categorical': 44159,
    'credit': 44089,
    'electricity': 44120,
    'electricity_categorical': 44156,
    'eyemovements': 44130,
    'eyemovements_categorical': 44156,
    'higgs': 44129,
    'house_16H': 44123,
    'jannis': 44131,
    'KDDCup06_upselling': 44186,
    'kdd_ipums_la_97-small': 44124,
    'magictelescope': 44125,
    'MiniBooNE': 44128,
    'phoneme': 44127,
    'pol': 44122,
    'rl': 44160,
    'road_safety': 44161,
    'wine': 44091,
```